### PR TITLE
Fix missing Virtual Switch in Config with Hyper-V

### PR DESCRIPTION
**Pull request by blueelvis**
## Description

This Pull request sets the VSwitch at the driver level after getting the best available switch from `chooseVirtualSwitch()`.

## Related issue(s)

This Pull Request fixes the issue mentioned over here - https://github.com/kubernetes/minikube/issues/5672

### CODE REVIEW

**Comment by blueelvis** [comment]
Could someone please review this PR once free? Thanks!

**Review by gbraad — APPROVED** [review]

**Comment by gbraad** [comment]
You addressed the wrong poeple to review the PR\

(Thomas is very busy with general Minikube issues, which is like Anders, very focussed on non-Windows platforms)

**Comment by gbraad** [comment]
Want to run a test ... so merging this later.

**Comment by blueelvis** [comment]
@gbraad - Thanks for the approval! I didn't know that you would be reviewing the PRs (I didn't see anyone I know in the `MAINTAINERS` file).

**Comment by gbraad** [comment]
We didn't modify these, but we might want to. Good point.

**Comment by blueelvis** [comment]
@gbraad - Could you please merge this PR? I don't have the permissions to merge it after your approval.

**Pull request merged**
